### PR TITLE
 Lockdown some gem versions for Hakiri/security

### PR DESCRIPTION
**Pull request by Fryguy**
Lockdown some gem versions for Hakiri/security

### CODE REVIEW

**Comment by Fryguy** [comment]
@bdunne @agrare Please review

**Comment by Fryguy** [comment]
See the second commit for the specific changes.

File: Gemfile [hunk 1]
```diff
@@ -1,46 +1,31 @@
...
+gem 'rails', '~> 5.1.5'
 
-# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem 'rails', '~> 5.1.0'
-# Use postgresql as the database for Active Record
-gem "pg", "~> 1.0", :require => false
-# Use Puma as the app server
-gem 'puma', '~> 3.0'
-# Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
 gem 'jbuilder', '~> 2.0'
```
Comment (Fryguy) [inline Gemfile]: Are we even using jbuilder?  Do we expect to?

Comment (agrare) [reply Gemfile]: I don't think so, @bdunne ?

Comment (Fryguy) [reply Gemfile]: Just asking for now...I can remove in a separate PR

Comment (bdunne) [reply Gemfile]: Nope

**Pull request merged**